static/js/global.js

```diff
@@ -0,0 +1,212 @@
+function  fillTableCell_Text (cellData) {
+    var td = document.createElement("td");
+    var txt = document.createTextNode(cellData);
+    td.appendChild(txt);
+    return td;
+}
+
+function Forecast() {
+    var self = this;
+
+    this.createForecastTable = function (data) {
+        if (data.hasOwnProperty('error_msg')) {
+            historyPage.loadHistoryPage();
+            $("#forecast_table").hide();
+            $("#error_msg").empty().show();
+
+            var error_div = document.getElementById('error_msg');
+            var txt = document.createTextNode(data['error_msg']);
+            error_div.appendChild(txt);
+
+        } else {
+            $("#error_msg").empty().hide();
+            $("#forecast_table_body").empty();
+            $("#forecast_table").show();
+
+            var table = document.getElementById("forecast_table_body");
+            var rows = document.createElement("tr");
+            rows.appendChild(fillTableCell_Text(data['city'] + data['country']));
+            rows.appendChild(fillTableCell_Text(data['min_temperature']));
+            rows.appendChild(fillTableCell_Text(data['max_temperature']));
+
+            table.appendChild(rows);
+        }
+    };
+
+    this.getForecastById = function (id) {
```

Comment:
The "getSomething" naming is usually reserved for functions that return a value. This function returns nothing; it starts a forecast ajax request. A better name could be picked.